ci-operator/step-registry/abi/install/bmc/abi-install-bmc-commands.sh

```
@@ -0,0 +1,520 @@
+        # Wait for all wipe Jobs to complete.
+        while true; do
+            kill -0 "${tPID}" 2>/dev/null || break
+            sleep 60
+            for jobId in "${jobIds[@]}"; do
+                {
+                    RedfishAPIcall "${bmcInfo}" "${bmcURL}" GET \
+                        "Managers/${bmcMgrId}/Jobs/${jobId}" |
+                    jq -e '
+                        .JobState | test("^Completed"; "i")
+                    '
+                } && {
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)